src/symmbasis.jl

```diff
@@ -295,12 +304,14 @@ end
 
 function evaluate_d!(dB, basis::SymmetricBasis,
                      AA::AbstractVector{<: Number}, dAA)
-   genmul!(dB, basis.A2Bmap, dAA,
+   genmul!(dB, basis.A2Bmap, dAA, 
            (a, b) -> _myreal1234(ACE.coco_o_daa(a, b), basis.real))
 end
 
+_scaling_get_a2b_vals(A2Bmap) =  [norm(A2Bmap[i,j].val) for i = 1:size(A2Bmap)[1], j = 1:size(A2Bmap)[2]]
```

Comment:
Another concern I have with this (which I should have spotted earlier is that you are converting a sparse matrix into a HUGE full matrix, it could  easily be 100k x 10k and more.  Can we keep is sparse?

Reply:
I think that’s the only thing that needs to be fixed before merging. Have you tried to simply do 
```julia
  norm.(A2Bmap)
```
?

Reply:
Maybe it doesn’t even need a separate function, just put it below.

Reply:
It works and I've checked their equivalence by code. Thanks for pointing this out as I should have tried that...